packages/esm-patient-orders-app/src/hooks/useOrderPrice.test.tsx

```diff
@@ -0,0 +1,101 @@
+import { act, renderHook } from '@testing-library/react';
+import { type FetchResponse, openmrsFetch } from '@openmrs/esm-framework';
+import { type OrderPriceData } from '../types/order';
+import { useOrderPrice } from './useOrderPrice';
+import { mockOrderPriceData } from '../../../../__mocks__/order-price-data.mock';
+
+const mockedOpenmrsFetch = jest.mocked(openmrsFetch);
+
+describe('useOrderPrice', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
```

Comment:
```suggestion
    jest.clearAllMocks();
```

Mocks get auto-cleared by default.